src/optimizer/query_planner.c

```diff
@@ -1128,7 +1128,7 @@ qo_plan_print_sort_spec_helper (PT_NODE * list, bool is_iscan_asc, FILE * f, int
 
       if (TP_TYPE_HAS_COLLATION (TP_DOMAIN_TYPE (list->info.sort_spec.pos_descr.dom))
 	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != LANG_SYS_COLLATION
-	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != (LANG_SYS_COLLATION + COLL_TS)
+	  && TP_DOMAIN_COLLATION (list->info.sort_spec.pos_descr.dom) != (LANG_SYS_COLLATION + COLL_TI)
```

Comment:
How about the following?
1) define LANG_GET_VARIATION_COLLATION (coll)
2) LANG_SYS_COLLATION + COLL_TI => LANG_GET_VARIATION_COLLATION (LANG_SYS_CODESET)
3) remove COLL_TI

Reply:
I add a macro "LANG_SYS_VARIATION" by defined as below:

#define LANG_SYS_VARIATION (LANG_SYS_COLLATION + COLL_TI)

Reply:
it seems better removing `#define COLL_TI 15`.

Reply:
I removed the macro COLL_TI